plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewMavenBasedAppEngineProjectWizardTest.java

```diff
@@ -20,8 +20,10 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 import com.google.cloud.tools.eclipse.appengine.facets.AppEngineStandardFacet;
+import com.google.cloud.tools.eclipse.test.util.ArrayAssertions;
```

Comment:
Not that I found.  I wanted something that would output the content of the array provided when the assertion failed.  It was helpful to understand problem markers, as you found on #2177 (https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2177#issuecomment-316660636)